samples/iolib/src/main/cpp/player/SimpleMultiPlayer.cpp

```diff
@@ -104,7 +106,14 @@ void SimpleMultiPlayer::setupAudioStream(int32_t numSampleBuffers, int32_t chann
     mChannelCount = channelCount;
     mSampleRate = sampleRate;
 
-    mSampleBuffers = new OneShotSampleBuffer[mNumSampleBuffers = numSampleBuffers];
+    mNumSampleBuffers = numSampleBuffers;
+    mSampleBuffers = new SampleBuffer*[mNumSampleBuffers];
```

Comment:
Consider using std::unique_ptr<SampleBuffer>

Reply:
Done, although I feel it makes the sample less clear.